BREAKING-CHANGE.tmp.md

```diff
@@ -0,0 +1,1280 @@
+# BREAKING: Map key changes
+
+An upcoming release of the **AWS SDK for Kotlin** will change the key type for several maps from strings into specific enumerations.
+
+# Release date
+
+This feature will ship with the **v1.0.80** release planned for **3/18/2024**.
```

Comment:
Are we doing a minor version bump?

Reply:
Oh yep, we are. I guess this will ship with **v1.1.0**.